src/main/resources/config.properties

```diff
@@ -0,0 +1,9 @@
+db.host=localhost
+db.user=root
+db.password=root
+db.name=gk_central
```

Comment:
An absolute nitpick - but do we consider gk_central public information? Maybe `curationDBName` or something? I don't know if it matters, but wanted to point it out